src/containers/InstanceListView.ts

```diff
@@ -0,0 +1,37 @@
+import { connect } from "react-redux";
```

Comment:
can we rename this to InstanceListContainer so we don't get confused with the component of the same name?